src/components/organisms/footer/footer.tsx

```diff
@@ -217,11 +217,11 @@ const Footer = () => {
   });
 
   return (
-    <footer className="flex justify-center w-full bg-[#134D82]">y
```

Comment:
This lil `y` is kinda hanging out in prod being cool, I am a bit sad to remove it tbh.

<img width="146" alt="Screenshot 2023-05-13 at 8 12 00 PM" src="https://github.com/TechIsHiring/techishiring-website/assets/8340719/d59782b6-c203-4c68-9bf6-0a0d325cd722">

Reply:
LOL Yeah, I must have added this by accident working on some other fixes and didn't catch it.

Ugh, I hate when that happens.